[ONNX] Move OptionalHasElement tests to runtime

titaiwangms opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #83599
* #83598
* #83186
* #83201
* __->__ #83185
* #83184

Fix #78119
Why:
1. Now that onnxruntime supports optional tensor execution, we should have its tests on runtime.
2. LoopNoneInput is skip right now in terms of the type inference error, more detail on https://github.com/onnx/onnx/issues/4424

titaiwangms committed:
move optionalhaselement tests to runtime

[ghstack-poisoned]

titaiwangms committed:
Update on "[ONNX] Move OptionalHasElement tests to runtime"


Why:
1. Now that onnxruntime supports optional tensor execution, we should have its tests on runtime.

[ghstack-poisoned]

titaiwangms committed:
Update on "[ONNX] Move OptionalHasElement tests to runtime"


Why:
1. Now that onnxruntime supports optional tensor execution, we should have its tests on runtime.

[ghstack-poisoned]

titaiwangms committed:
Update on "[ONNX] Move OptionalHasElement tests to runtime"


Why:
1. Now that onnxruntime supports optional tensor execution, we should have its tests on runtime.
2. LoopNoneInput is skip right now in terms of the type inference error, more detail on https://github.com/onnx/onnx/issues/4424

[ghstack-poisoned]

titaiwangms committed:
Update on "[ONNX] Move OptionalHasElement tests to runtime"


Why:
1. Now that onnxruntime supports optional tensor execution, we should have its tests on runtime.
2. LoopNoneInput is skip right now in terms of the type inference error, more detail on https://github.com/onnx/onnx/issues/4424

[ghstack-poisoned]

titaiwangms committed:
Update on "[ONNX] Move OptionalHasElement tests to runtime"


Fix #78119
Why:
1. Now that onnxruntime supports optional tensor execution, we should have its tests on runtime.
2. LoopNoneInput is skip right now in terms of the type inference error, more detail on https://github.com/onnx/onnx/issues/4424

[ghstack-poisoned]

titaiwangms committed:
Update on "[ONNX] Move OptionalHasElement tests to runtime"


Fix #78119
Why:
1. Now that onnxruntime supports optional tensor execution, we should have its tests on runtime.
2. LoopNoneInput is skip right now in terms of the type inference error, more detail on https://github.com/onnx/onnx/issues/4424

[ghstack-poisoned]

titaiwangms committed:
Update on "[ONNX] Move OptionalHasElement tests to runtime"


Fix #78119
Why:
1. Now that onnxruntime supports optional tensor execution, we should have its tests on runtime.
2. LoopNoneInput is skip right now in terms of the type inference error, more detail on https://github.com/onnx/onnx/issues/4424

[ghstack-poisoned]

titaiwangms committed:
Update on "[ONNX] Move OptionalHasElement tests to runtime"


Fix #78119
Why:
1. Now that onnxruntime supports optional tensor execution, we should have its tests on runtime.
2. LoopNoneInput is skip right now in terms of the type inference error, more detail on https://github.com/onnx/onnx/issues/4424

[ghstack-poisoned]

shubhambhokare1 commented:
LGTM 👍

shubhambhokare1 reviewed (approved):
LGTM! 👍 Can we merge this?

AllenTiTaiWang commented:
> LGTM! 👍 Can we merge this?

ghstack comes in order. Let me finish previous one first.

titaiwangms committed:
Update on "[ONNX] Move OptionalHasElement tests to runtime"


Fix #78119
Why:
1. Now that onnxruntime supports optional tensor execution, we should have its tests on runtime.
2. LoopNoneInput is skip right now in terms of the type inference error, more detail on https://github.com/onnx/onnx/issues/4424

[ghstack-poisoned]
